### PR TITLE
Added security tests and dependency scan

### PR DESCRIPTION
**Pull request by omerlh**

### CODE REVIEW

**Comment by omerlh** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Checklist of the most important security countermeasures when designing, testing
 - [ ] Audit your design and implementation with unit/integration tests coverage.
 - [ ] Use a code review process and disregard self-approval.
 - [ ] Ensure that all components of your services are statically scanned by AV software before push to production, including vendor libraries and other dependencies.
+- [ ] Continuesly run security tests (static/dynamic analysis) on your code.
```
Comment (silleknarf) [inline README.md:63]:
```suggestion
- [ ] Continuously run security tests (static/dynamic analysis) on your code.
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Checklist of the most important security countermeasures when designing, testing
...
+- [ ] Check your dependency (both software and OS) for known vulnerabilities.
```
Comment (silleknarf) [inline README.md:64]:
```suggestion
- [ ] Check your dependencies (both software and OS) for known vulnerabilities.
```

**Comment by Maikuolan** [comment]
Merged. Thanks for the contribution. :+1: 😊

**Pull request merged**